docs/tutorials/random.rst

```diff
@@ -1,12 +1,11 @@
-Random
+Losowość
```

Comment:
Ten plik już został raz przetłumaczony i jest w osobnym pull requescie.

Reply:
OK, to był jakiś błąd w wysyłaniu commita ze złego brancha...